src/functionaltests/java/com/ericsson/ei/subscriptions/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -0,0 +1,229 @@
+        objMapper = new ObjectMapper();
+
+        try {
+            aggrObj = FileUtils.readFileToString(new File(aggregatedObjJsonPath), "UTF-8");
+            missedNotificationObj = FileUtils.readFileToString(new File(missedNotificationJsonPath), "UTF-8");
+
+        } catch (IOException e) {
+            LOGGER.error("Failed to open test json files for test. Error message\n: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+    
+    @Given("^Aggregated object is created$")
+    public void aggregated_object_is_created() throws Throwable {
+        LOGGER.debug("Creating aggregated object in MongoDb");
+        assertEquals(true, createDocumentInMongoDb(eiDatabseName, aggrCollectionName, aggrObj));
+    }
+
+    @Given("^Missed Notification object is created$")
+    public void missed_notification_object_is_created() throws Throwable {
+        LOGGER.debug("Missed Notification object has been created in MongoDb");
+        assertEquals(true, createDocumentInMongoDb(missedNotificationDatabaseName, missedNotificationCollectionName, missedNotificationObj));
+    }
+
+    @Then("^Perform valid query on newly created Aggregated object")
+    public void perform_valid_query_on_newly_created_aggregated_object() throws Throwable {
+        String expectedTestCaseStartedEventId = "cb9d64b0-a6e9-4419-8b5d-a650c27c59ca";
+        String entryPoint = "/queryAggregatedObject";
+        String queryRequest = "{\"id\":\"6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43\"}";
+        String documentId = getDocumentIdBasedOnQuery(eiDatabseName, aggrCollectionName, queryRequest);
+        LOGGER.debug("Got AggregateObject actual DocumentId after querying MongoDB: " + documentId);
+        mvcResult = mockMvc.perform(get(entryPoint)
+                .param("ID", documentId)
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andReturn();
+        LOGGER.debug("Response of /queryAggregatedObject RestApi, Status Code: " + mvcResult.getResponse().getStatus() +
+                           "\nResponse: " + mvcResult.getResponse().getContentAsString());
+        
+        JsonNode jsonNodeResult = new ObjectMapper().readValue(mvcResult.getResponse().getContentAsString(), JsonNode.class);
+        JsonNode responseEntityNode = jsonNodeResult.get("responseEntity");
+        String responseEntityNodeFormatted = responseEntityNode.asText().replace("[", "").replace("]", "");
+        JsonNode responseEntityFormattedJsonNode = new ObjectMapper().readValue(responseEntityNodeFormatted, JsonNode.class);
+        
+        LOGGER.debug("AggregatedObject from Response: " + responseEntityFormattedJsonNode.toString());
+       
+        String actualTestCaseStartedEventId = responseEntityFormattedJsonNode.get("testCaseExecutions").get("testCaseStartedEventId").asText();
+        assertEquals(HttpStatus.OK.toString(), Integer.toString(mvcResult.getResponse().getStatus()));
```

Comment:
HttpStatus.OK.value()- returns int just use this and than unneeded convert int to string